### ISSUE TITLE
Import `QtCore.Qt` failure -  `AttributeError: Cannot reassign members.`

### ISSUE THREAD

**Issue by owillebo** [spyder-ide/qtpy#377]
Running in a venv;
```
Python 3.9.13 (tags/v3.9.13:6de2ca5, May 17 2022, 16:36:42) [MSC v.1929 64 bit (AMD64)] on win32
packaging==21.3
pip==22.0.4
pyparsing==3.0.9
PySide6==6.4.0
PySide6-Addons==6.4.0
PySide6-Essentials==6.4.0
QtPy==2.2.1
setuptools==58.1.0
shiboken6==6.4.0
```

from qtpy.QtCore import Qt

results in;
```
Traceback (most recent call last):
  File "C:\Users\XXXXXXXX\Desktop\okko.py", line 1, in <module>
    \ufefffrom qtpy.QtCore import Qt
  File "C:\Users\XXXXXXXX\Desktop\okko\.venv\lib\site-packages\qtpy\QtCore.py", line 116, in <module>
    Qt.MidButton = Qt.MouseButton.MiddleButton = Qt.MiddleButton
  File "shibokensupport/enum_310.py", line 529, in __setattr__
AttributeError: Cannot reassign members.
```
Qt.MouseButton.MiddleButton exists before the assignment.

**Comment by dalthviz** [comment]
Hi @owillebo thank you for the feedback! This looks like a duplicate of #373 and it was already fixed with #374 

Just in case, we are currently checking what other elements could need an update due to PySide6 6.4.0 release at #375 and #376 . For the moment, to prevent this issue, you will need to downgrade PySide 6 to a 6.3.x version.

Let us know if the info above helps!

**Comment by mchlnix** [comment]
Had this issue with Python 3.11 and downgrading PySide is not an option, since 6.4.0.1 is the only one supporting 3.11 atm. Not an issue, just so you know in case it comes up.

**Comment by dalthviz** [comment]
Hi @mchlnix thank you for the feedback! We just released QtPy 2.3.0, could you test if it works as expected on your setup (Python 3.11 and Pyside6 6.4.0.1)? Let us know!

**Comment by mchlnix** [comment]
Works for me :+1:

**Comment by owillebo** [comment]
Works like a charm!

**Comment by dalthviz** [comment]
Awesome, thank you for the feedback @mchlnix @owillebo ! I'm closing this one then as a duplicate of #373 (fix available with QtPy v2.3.0)
